ubuntu-simple/README.md

```diff
@@ -6,166 +6,208 @@ keywords: weave, docker, containers, networking, software defined networking, do
-    aa:3c:6c:5f:fa:31 -> []
-    Direct Peers:
-    Reconnects:
-
-    Allocator range [10.128.0.0-10.192.0.0)
-    Owned Ranges:
-      10.128.0.0 -> b2:6d:8c:74:4a:31 (weave-gs-01) (v3)
-    Allocator default subnet: 10.128.0.0/10
-
-    weave DNS 1.0.1
-    Listen address :53
-    Fallback DNS config &{[10.0.2.3] [overplay] 53 1 5 2}
-
-    Local domain weave.local.
-    Interface &{18 65535 ethwe 4e:60:c7:bd:20:0c up|broadcast|multicast}
-    Zone database:
-
-## Our Hello World Service ##
-
-Next you will use Weave to run a Docker image containing an Apache webserver.  Details on how this container was created using docker are available [here](https://github.com/weaveworks/guides/blob/master/ubuntu-simple/DockerfileREADME.md).
+    e3fba94a35fc        weaveworks/weavedns:1.0.2   "/home/weave/weavedn   57 seconds ago      Up 56 seconds       10.1.42.1:53->53/udp                             weavedns
+    dd3878af6307        weaveworks/weave:1.0.2      "/home/weave/weaver    16 minutes ago      Up 16 minutes       0.0.0.0:6783->6783/tcp, 0.0.0.0:6783->6783/udp   weave
+~~~
+
+View the peered network by running `weave status`
+
+~~~bash
+sudo weave status
+
+weave router 1.0.2
+Our name is 72:24:c2:00:01:5a(weave-gs-02)
+Encryption off
+Peer discovery on
+Sniffing traffic on &{10 65535 ethwe ee:08:d8:0e:ec:cb up|broadcast|multicast}
+MACs:
+6a:0d:0a:55:3f:a1 -> 72:24:c2:00:01:5a(weave-gs-02) (2015-08-21 20:09:26.761668678 +0000 UTC)
+8a:2b:ec:2a:b7:0f -> a6:e9:63:3e:d2:dd(weave-gs-01) (2015-08-21 20:07:45.075079927 +0000 UTC)
+f2:86:dc:cc:9a:43 -> 72:24:c2:00:01:5a(weave-gs-02) (2015-08-21 20:09:06.151707913 +0000 UTC)
+Peers:
+a6:e9:63:3e:d2:dd(weave-gs-01) (v2) (UID 13063150984085781788)
+   -> 72:24:c2:00:01:5a(weave-gs-02) [172.17.8.102:51443]
+72:24:c2:00:01:5a(weave-gs-02) (v2) (UID 10318206013728323935)
+   -> a6:e9:63:3e:d2:dd(weave-gs-01) [172.17.8.101:6783]
+Routes:
+unicast:
+72:24:c2:00:01:5a -> 00:00:00:00:00:00
+a6:e9:63:3e:d2:dd -> a6:e9:63:3e:d2:dd
+broadcast:
+a6:e9:63:3e:d2:dd -> []
+72:24:c2:00:01:5a -> [a6:e9:63:3e:d2:dd]
+Direct Peers: 172.17.8.101
+Reconnects:
+
+Allocator range [10.128.0.0-10.192.0.0)
+Owned Ranges:
+  10.128.0.0 -> 72:24:c2:00:01:5a (weave-gs-02) (v5)
+  10.160.0.0 -> a6:e9:63:3e:d2:dd (weave-gs-01) (v2)
+Allocator default subnet: 10.128.0.0/10
+
+weave DNS 1.0.2
+Listen address :53
+Fallback DNS config &{[10.0.2.3] [] 53 1 5 2}
+
+Local domain weave.local.
+Interface &{14 65535 ethwe 6a:0d:0a:55:3f:a1 up|broadcast|multicast}
+Zone database:
+
+~~~
+
+## Deploying the Hello World Service ##
+
+Next, use Weave to run a Docker image containing an Apache webserver.  Details on how this container was created using docker are available [here](https://github.com/weaveworks/guides/blob/master/ubuntu-simple/DockerfileREADME.md).
 
 On `weave-gs-01` run
 
-    sudo weave run --name=web1 -t -i fintanr/weave-gs-simple-hw
+~~~bash
+sudo weave run --name=web1 -t -i fintanr/weave-gs-simple-hw
```

Comment:
This should use `docker run` instead, for which you will need to do `eval $(weave proxy-env)` first...